setup.py

```diff
@@ -14,6 +14,8 @@
     # Project uses reStructuredText, so ensure that the docutils get
     # installed or upgraded on the target machine
     install_requires=[
+        "cryptography",
```

Comment:
@chmoder, Could you freeze these newly added libraries? I think there might be some problems when someone uses this library. What do you think?

Reply:
Great suggestions!  I pinned the version of pyJWT to the latest or anything that comes after.  And pyJWT has required cryptography>=3.4.0 for ~2 years now so I pinned ours to that dependency as well.